docs/blog/newsletter-february-2018.rst

```diff
@@ -0,0 +1,121 @@
+that are good to write for.
+
+Not only is this a good resource for any freelance folks out there, but it's also a great opportunity to help build it up as a resource. 
+`You can find the list on Github here <https://github.com/ChrisChinchilla/awesome-people-to-write-for>`_. 
+Have a look, give it a fork, and add any publications that you've had good experiences with as a contributor.
+
+*Have a question or want to chat about the list? 
+Pop it in* `#general <https://writethedocs.slack.com/messages/general>`_ *and tag @chrisward on Slack.*
+
+*************************
+Upcoming community events
+*************************
+
+Just in case we didn't express enough excitement in the intro, 
+here's your friendly reminder that tickets for the Portland conference are going fast! 
+Come join us **May 6-8 at the Crystal Ballroom**! 
+`Head over to the conference site to buy yours! <http://www.writethedocs.org/conf/portland/2018/tickets/>`_
+
+And to tide you over until then, we've got loads of meetups happening all over the world this month!
+
+February 7 – Austin, TX, USA – `Monthly meetup <https://www.meetup.com/WriteTheDocs-ATX-Meetup/events/kqbbznyxcbnb/>`_
+
+February 8 – Melbourne, Australia – `Getting devs to write good docs | Making Imposter Syndrome work for you <https://www.meetup.com/Write-the-Docs-Australia/events/245510804/>`_
+
+February 12 – Berlin, Germany – `When in doubt, use C4 <https://www.meetup.com/Write-The-Docs-Berlin/events/246990715/>`_
+
+February 13 – San Diego, CA, USA – `Swagger UI and the Open API Specification with Tom Johnson <https://www.meetup.com/STC-San-Diego/events/242200388/>`_
+
+February 19 – Amsterdam, Netherlands – `Testing Documentation <https://www.meetup.com/Write-The-Docs-Amsterdam/events/246609720/>`_
+
+February 20 – Dublin, Ireland – `An introduction to effective corporate blogging <https://www.meetup.com/Write-The-Docs-Ireland/events/246089836/>`_
+
+February 21 – New York City, NY, USA – `Discussion: Managing Technical Writers <https://www.meetup.com/WriteTheDocsNYC/events/247397487/>`_
+
+February 22 – Boise, ID, USA – `First meetup of 2018 <https://www.meetup.com/Write-the-Docs-Boise/events/246900941/>`_
+
+February 22 – Sydney, Australia – `How to build an IA | Q&A: All things tech documentation <https://www.meetup.com/Write-the-Docs-Australia/events/245510988/>`_
+
+February 22 – Cambridge, UK – `Markup languages: a brief history <https://www.meetup.com/Write-The-Docs-Cambridge/events/246750191/>`_
+
+March 1 – Los Angeles, CA, USA – `First Pasadena Meetup <https://www.meetup.com/Write-the-Docs-Los-Angeles/events/247164479/>`_
+
+March 6 – Portland, OR, USA – `Monthly Meetup <https://www.meetup.com/Write-The-Docs-PDX/>`_
+
+*NOTE: If you run a meetup group and you're not seeing your events show up here in this calendar, drop me a DM (@kobrien042) on Slack to let me know. The Meetup site search is not, like, the best.*
```

Comment:
Do we wanna run `The Meetup site search is not, like, the best.*` ?

Reply:
kinda – it is not the best

Reply:
but I didn't